azure-cli/appservice/deploy_AppServicePlan.ps1

```diff
@@ -0,0 +1,72 @@
+function Deploy-AppServicePlan {
+  param (
+    [Parameter(Mandatory = $true)]
+    [ValidateNotNullOrEmpty()]
+    [string]
+    $AppServicePlanName,
+
+    [Parameter(Mandatory = $true)]
+    [ValidateNotNullOrEmpty()]
+    [string]
+    $ResourceGroupName,
+
+    [Parameter(Mandatory = $false)]
+    [ValidateNotNullOrEmpty()]
+    [ValidateSet('DevTest', 'Production')]
+    [string]
+    $EnvironmentType = "DevTest",
+
+    [Parameter(Mandatory = $false)]
+    [ValidateNotNullOrEmpty()]
+    [string]
+    $Location = "westeurope",
+  
+    [Parameter(Mandatory = $false)]
+    [string[]] $ResourceTags = @()
+  )  
+  Write-Host "Provision app service plan '$AppServicePlanName'" -ForegroundColor DarkGreen
+
+  $sku = 'S1'
```

Comment:
Would it make sense to move this descision out of the function and inject the value as an input parameter?

Reply:
Yes